config/environments/staging.rb

```diff
@@ -37,7 +37,7 @@
     email: {
       sender_address: %{"Notifier" <no-reply@musc.edu>}, 
       exception_recipients: ENV.fetch('EXCEPTION_NOTIFICATION_RECIPIENTS'), 
-      email_prefix: "[RMID-#{Rails.env.upcase}-ERROR]"
+      email_prefix: "[RMID-#{Rails.root.to_s}-ERROR]"
```

Comment:
What does this give you?  I think the request was to have something like `research-master-staging.musc.edu` added instead of the ENV.  I feel like this would give you the path `/var/www/rails/research-master/releases/ABC123`